src/components/ReadMore/_readMore.scss

```diff
@@ -0,0 +1,57 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-read-more {
+  align-items: center;
+  display: flex;
```

Comment:
does this actually have to be a flex container?  it seems like it only has the one child

Reply:
it actually has two – the `CollapsibleContent` and the overlay wrapper. When closed, the overlay wrapper is absolutely positioned, but when expanded it is in the same flex body as the content (at which point the `gap` style below this adds spacing between the two)